apps/demos/Demos/FileUploader/CustomDropzone/Vue/App.vue

```diff
@@ -69,19 +69,19 @@ function onDropZoneEnter({ component, dropZoneElement, event }: DxFileUploaderTy
     const draggedFileExtension = `.${items[0].type.replace(/^image\//, '')}`;
 
     const isSingleFileDragged = items.length === 1;
-    const isValidFileExtension = allowedFileExtensions.includes(draggedFileExtension);
+    const isValidFileExtension = !!allowedFileExtensions?.includes(draggedFileExtension);
```

Comment:
Is this mandatory? includes returns boolean; if allowedFileExtensions is undefined, then this is also acceptable here.
```suggestion
    const isValidFileExtension = allowedFileExtensions?.includes(draggedFileExtension);
```

Reply:
allowedFileExtensions? may be undefined but type must be boolean